### PR TITLE
eco model relations revamp

### PR DESCRIPTION
**Pull request by ricerb**
I modified the eco schema as proposed in the diagram below and modified fields according to Caroline's suggestions.
 - Also added a duration value float field in the Cause table

image

### CODE REVIEW

**Comment by ricerb** [comment]
@shapiromatron I think it might make sense to change the inline admin form for the Results entries given the schema changes, but I didn't edit the forms beyond making them work with the table name and relationship changes.

**Comment by ricerb** [comment]
I implemented schema changes, modifying factors tags, and other minor changes. @shapiromatron this is ready for your review!

**Review by shapiromatron — APPROVED** [review]
Looks good! I made some changes, mostly to the admin views, and added a a few more `related_name` values to the models

File: hawc/constants.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class AuthProvider(str, Enum):
 
 class FeatureFlags(BaseModel):
     THIS_IS_AN_EXAMPLE: bool = True
-    ENABLE_ECO: bool = False
+    ENABLE_ECO: bool = True
```
[the remark this answers is not in the document]
Comment (ricerb) [reply hawc/constants.py]: Thanks @shapiromatron! Glad to see this moving forward.

**Pull request merged**